service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/util/ReferenceUtil.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.adaptors.oneoneone.cda.report.util;
+
+import java.util.Arrays;
+import java.util.function.Predicate;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.hl7.fhir.dstu3.model.DomainResource;
+import org.hl7.fhir.dstu3.model.Reference;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class ReferenceUtil {
+
+    /**
+     * @param type DomainResource type
+     * @return Predicate which returns true if the FHIR Reference matches the name of the type <strong>exactly</strong>.
```

Comment:
There is no `exactly` in code

Reply:
I think this comment means that it returns true if the name matches exactly, not literally the word `exactly`.